### PR TITLE
ci: Add OwlBot configuration

### PR DESCRIPTION
**Pull request by kweinmeister**

### CODE REVIEW

File: .github/workflows/.OwlBot.lock.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Google LLC
```
Comment (parthea) [inline .github/workflows/.OwlBot.lock.yaml:1]: `.github/workflows/.OwlBot.lock.yaml`->`.github/.OwlBot.lock.yaml`

File: .github/workflows/.OwlBot.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Google LLC
```
Comment (parthea) [inline .github/workflows/.OwlBot.yaml:1]: `.github/workflows/.OwlBot.yaml`->`.github/.OwlBot.yaml`

**Review by parthea — COMMENTED** [review]
Added one observation

File: .github/workflows/.OwlBot.lock.yaml (hunk 1 above)
Comment (kweinmeister) [reply .github/workflows/.OwlBot.lock.yaml]: Oops! Fixed.

**Review by parthea — APPROVED** [review]

**Pull request merged**